modules/tinymce/src/core/main/ts/api/util/Tools.ts

```diff
@@ -113,11 +113,13 @@ const makeMap = (items, delim?, map?) => {
 
 /**
  * JavaScript does not protect hasOwnProperty method, so it is possible to overwrite it. This is
- * object independent version.
+ * an object independent version.
+ * Checks if the input object "<code>obj</code>" has the property "<code>prop</code>".
  *
- * @param {Object} obj
- * @param {String} prop
- * @returns {Boolean}
+ * @method hasOwnProperty
+ * @param {Object} obj Object to test.
+ * @param {String} prop Property to test <code>obj</code> for.
+ * @returns {Boolean} true if the object "<code>obj</code>" has the property "<code>prop</code>".
```

Comment:
```suggestion
 * @param {Object} obj Object to check if the property exists.
 * @param {String} prop Name of a property on the object.
 * @returns {Boolean} true if the object has the specified property.
```
"test" isn't really clear in this case and could be confused with other forms of testing. Also we really shouldn't be using HTML in property descriptions like this.